src/scripts/main.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('tbody');
+const addRow = document.querySelector('.append-row');
+const delRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const delColumn = document.querySelector('.remove-column');
+
+const creatTr = () => {
+    const element = table.children[0].cloneNode(true)
+    table.append(element);
+}
+
+const createTd = () => {
+    for(const elem of table.children) {
+       const element = elem.lastElementChild.cloneNode(true);
+        elem.append(element);
+    }
+}
+
+let colum = 4;
+let row = 4;
+
+
+addRow.onclick = () => {
+    creatTr();
+    if (row === 9) {
+        addRow.disabled = true;
+        row++;
+        return;
+    } else if (row === 2) {
+        delRow.disabled = false;
```

Comment:
let's move disabling  outside the function to get rid of the same check in each listener

Reply:
if I drag over the function area, then the check will not work because it works when clicked, maybe I don't understand what you mean :)